examples/wasm-shell/components/wasm-terminal.tsx

```diff
@@ -52,17 +56,24 @@ const commands = {
   arg: argtestUrl,
   clang: clangUrl,
   sqlite: sqliteUrl,
-  rsign:
-    "https://registry-cdn.wapm.io/contents/jedisct1/rsign2/0.5.4/rsign.wasm",
+  rsign: rsignUrl,
+  uutils: uutilsUrl,
   callback: (args: string[], stdin: string) => {
     return Promise.resolve(
       `Callback Command Working! Args: ${args}, stdin: ${stdin}`
     );
   }
 };
 
+const commandBinaryCache: any = {};
```

Comment:
Thanks for adding the cache! Is this up to date on the latest master? I thought I added one alread? 🤔

Reply:
This is yours! :) (I didn't add it myself)

Reply:
(check the commits on this PR, two are yours)